docs/compatibility.md

```diff
@@ -8,9 +8,9 @@ nav_order: 6
 
 ## Ruby & Rails
 
-ViewComponent is [supported natively](https://edgeguides.rubyonrails.org/layouts_and_rendering.html#rendering-objects) in Rails 6.1, and compatible with Rails 5.2+ via an included [monkey patch](https://github.com/viewcomponent/view_component/blob/main/lib/view_component/render_monkey_patch.rb).
+View Component supports all actively supported versions of Ruby (3.0+) and Ruby on Rails(6.1+) and is tested against a combination of these versions of ruby and rails
 
-ViewComponent is tested for compatibility [with combinations of](https://github.com/viewcomponent/view_component/blob/22e3d4ccce70d8f32c7375e5a5ccc3f70b22a703/.github/workflows/ruby_on_rails.yml#L10-L11) Ruby v2.5+ and Rails v5.2+. Ruby 2.4 is likely compatible, but is no longer tested.
+Rails >= 6.0 and Ruby >= 2.7 may still work with view_component, but this configuration is not actively supported and any issues submitted against unsupported versions of Ruby or Rails have no guarantee of being fixed as they may eb arising from issues from these unsupported versions
```

Comment:
```suggestion
While EOL (end-of-life) versions of Ruby (>= 2.7) and Ruby on Rails (> 6.0) may still work with ViewComponent, they are not actively supported and we do not active test the gem against them. We will still accept patches on a case-by-case basis to support older Ruby and Ruby on Rails versions based on the complexity and maintenance burden. Please open an issue before submitting a Pull Request that addresses issues with EOL Ruby/Ruby on Rails versions.
```

Thoughts on this? This is how I was imagining we'd handle this situation since it adds some reasonable flexibility without keeping us stuck maintaining EOL dependencies.

Reply:
I like this @BlakeWilliams, I struggle with the wording sometimes, but this sums it up just right